tests/test_det_city_planning.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+from datetime import date, time
+
+
+from tests.utils import file_response
+from city_scrapers.spiders.det_city_planning import DetCityPlanningSpider
+
+
+test_response = file_response('files/det_city_planning.html', url='https://www.detroitmi.gov/Government/Boards/City-Planning-Commission-Meetings')
+spider = DetCityPlanningSpider()
+parsed_items = [item for item in spider.parse(test_response) if isinstance(item, dict)]
```

Comment:
Here it might be a good idea to sort these based on date, similar to https://github.com/City-Bureau/city-scrapers/pull/481/files#diff-e8f00c59a44be8fe98cff4fc44aaef24R112. Since the spider the meetings go into a dictionary before iterating over them (i.e. can't guarantee order).

Reply:
Done.